hvantk/tests/test_llm_interface.py

```
@@ -0,0 +1,384 @@
+import pytest
+from unittest.mock import MagicMock, patch
+import hail as hl
+import os
+
+from hvantk.utils.llm_interface import natural_language_query, LLMInterface, LLMConfigError
```

Ruff (0.11.9)

`hvantk.utils.llm_interface.LLMConfigError` imported but unused

Remove unused import: `hvantk.utils.llm_interface.LLMConfigError`

(F401)